Add Cond Access info (#3854)
MicrosoftDocs/OfficeDocs-SharePoint: SharePoint/SharePointOnline/allow-syncing-only-on-specific-domains.md

   > Make sure to add the domain GUID of the computer domain membership. If users are in a separate domain, only the domain GUID that the computer account is joined to is required.

   > [!IMPORTANT]

>    > This setting is only applicable to Active Directory domains. It does not apply to Azure AD domains. If you have devices which are only Azure AD joined, consider using a [Conditional Access Policy](/azure/active-directory/conditional-access/overview) instead, see also [Enable conditional access support in the OneDrive sync app](enable-conditional-access.md).

Reviewer: ```suggestion
   > This setting is only applicable to Active Directory domains. It does not apply to Azure AD domains. If you have devices that are only Azure AD joined, consider using a [Conditional Access Policy](/azure/active-directory/conditional-access/overview) instead, For details, see [Enable conditional access support in the OneDrive sync app](enable-conditional-access.md).
```

>    > This setting is only applicable to Active Directory domains. It does not apply to Azure AD domains. If you have devices that are only Azure AD joined, consider using a [Conditional Access Policy](/azure/active-directory/conditional-access/overview) instead, For details, see [Enable conditional access support in the OneDrive sync app](enable-conditional-access.md).

Reviewer: ```suggestion
   > This setting is only applicable to Active Directory domains. It does not apply to Azure AD domains. If you have devices that are only Azure AD joined, consider using a [Conditional Access Policy](/azure/active-directory/conditional-access/overview) instead. For details, see [Enable conditional access support in the OneDrive sync app](enable-conditional-access.md).
```
